lib/grpc_reflection/service/state.ex

```diff
@@ -56,8 +58,10 @@ defmodule GrpcReflection.Service.State do
   def lookup_symbol("." <> symbol, state), do: lookup_symbol(symbol, state)
 
   def lookup_symbol(symbol, %__MODULE__{symbols: symbols}) do
-    if Map.has_key?(symbols, symbol) do
-      {:ok, symbols[symbol]}
+    {_, root_symbol} = Util.get_package_and_root_symbol(symbol)
```

Comment:
> Anything that could be discovered here should instead be discovered during build and explicitly stored in the State object.

I got your point. It would be better to add a new nested map to the `state` struct, like this.
`%GrpcReflection.Service.State{root_symbols: %{}}`